lib/Varien/Data/Tree/Db.php

```diff
@@ -114,11 +114,17 @@ public function __construct($connection, $table, $fields)
         $this->_select->from($this->_table, array_values($fields));
     }
 
+    /**
+     * @return Zend_Db_Select
```

Comment:
Varien_Db_Select ? and in the property too?

Reply:
I'm pretty sure if xdebuged this .... i'll check again.

Reply:
It is ok. Please check `__construct()` ...

`$this->_select  = $this->_conn->select();` returns ....

```
    /**
     * Creates and returns a new Zend_Db_Select object for this adapter.
     *
     * @return Zend_Db_Select
     */
    public function select()
    {
        return new Zend_Db_Select($this);
    }
```